### PR TITLE
build-capi

### PR DESCRIPTION
**Pull request by Horta**
I've changed the name from build_capi to build-capi. I'm normalizing all my packages...

### CODE REVIEW

**Comment by Horta** [comment]
Thus, I would like to ask for the removal of https://github.com/conda-forge/build_capi-feedstock in favor of this new one that will become https://github.com/conda-forge/build-capi-feedstock

**Comment by Horta** [comment]
Hello guys. Is there any problem with circleci?

Also, I'm the author of this package and recipe but I don't see how to remove https://github.com/conda-forge/build_capi-feedstock by myself, so I suppose I need to ask here, right?

Thanks!

**Comment by jakirkham** [comment]
There is. As a workaround, you could enable CircleCI on your fork, but I don't think it is necessary in the case of this recipe.

xref: https://github.com/conda-forge/status/issues/30

File: recipes/build-capi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
Comment (jakirkham) [inline recipes/build-capi/meta.yaml]: Please reset to `0`.

**Comment by jakirkham** [comment]
Thanks @Horta .

**Pull request merged**